include/grpcpp/impl/codegen/call.h

```diff
@@ -623,42 +1156,66 @@ class CallOpSet : public CallOpSetInterface,
                   public Op5,
                   public Op6 {
  public:
-  CallOpSet() : cq_tag_(this), return_tag_(this), call_(nullptr) {}
-
+  CallOpSet() : cq_tag_(this), return_tag_(this) {}
   // The copy constructor and assignment operator reset the value of
   // cq_tag_ and return_tag_ since those are only meaningful on a specific
   // object, not across objects.
```

Comment:
It looks like the above comment needs to get expanded now

Reply:
done!